### PR TITLE
EZP-30346: Broken notifications with html 

### PR DESCRIPTION
**Pull request by ViniTou**
>Jira : https://jira.ez.no/browse/EZP-30346

Related:
https://github.com/ezsystems/ezplatform-admin-ui/pull/930

### CODE REVIEW

File: src/modules/sub-items/sub.items.module.js [hunk 1]
```diff
@@ -455,28 +455,42 @@ export default class SubItemsModule extends Component {
...
-                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
+                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. {{ placeholder }} Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
                 {
                     notMovedCount: notMovedLocations.length,
                     totalCount: movedLocations.length + notMovedLocations.length,
                 },
                 'sub_items'
             );
+            const rawPlaceholdersMap = {
+                placeholder: Translator.trans(
```
Comment (dew326) [inline src/modules/sub-items/sub.items.module.js]: I would use a more explanatory key name. What if we will have to add the second placeholder?

**Review by dew326 — CHANGES_REQUESTED** [review]

**Comment by ViniTou** [comment]
@dew326 
should look better now.

**Review by dew326 — APPROVED** [review]

File: src/modules/sub-items/sub.items.module.js [hunk 2]
```diff
@@ -455,28 +455,42 @@ export default class SubItemsModule extends Component {
...
+                    {
+                        locationName: eZ.helpers.text.escapeHTML(location.ContentInfo.Content.Name),
+                        locationHref: this.props.generateLink(location.id),
+                    },
+                    'sub_items'
+                ),
+            };
 
-            window.eZ.helpers.notification.showSuccessNotification(message);
+            window.eZ.helpers.notification.showSuccessNotification(message, () => {}, rawPlaceholdersMap);
```
Comment (tischsoic) [inline src/modules/sub-items/sub.items.module.js:493]: Nitpick: instead of passing `() => {}` we can pass `null`.

**Review by tischsoic — APPROVED** [review]

**Review by katarzynazawada — APPROVED** [review]

**Comment by lserwatka** [comment]
@ViniTou could you rebase?

**Comment by ViniTou** [comment]
@lserwatka 
done

**Comment by lserwatka** [comment]
Could you merge it up?

**Pull request merged**